interface/main/finder/patient_select.php

```diff
@@ -290,7 +290,10 @@ function submitList(offset) {
    </a>
    &nbsp;&nbsp;
 <?php } ?>
-   <?php echo ($fstart + 1) . htmlspecialchars( " - $fend of $count", ENT_NOQUOTES); ?>
+  <?php
+   $countStatement =  " - " . $fend . xl('of') .  $count;
```

Comment:
need to work in spacing:
``````
$countStatement =  " - " . $fend . " " . xl('of') . " " .  $count;